pkg/internal/apis/certmanager/validation/certificaterequest.go

```diff
@@ -200,3 +280,12 @@ func ensureCertSignIsSet(list []cmapi.KeyUsage) []cmapi.KeyUsage {
 
 	return append(list, cmapi.UsageCertSign)
 }
+
+func getCertificateRequestCondition(conds []cmapi.CertificateRequestCondition, conditionType cmapi.CertificateRequestConditionType) *cmapi.CertificateRequestCondition {
+	for _, cond := range conds {
+		if cond.Type == conditionType {
+			return &cond
+		}
+	}
+	return nil
+}
```

Comment:
Looks like all the new code in this file is covered:
![image](https://user-images.githubusercontent.com/978965/111463177-3352e400-8717-11eb-8a6f-8c2381e0dbbf.png)